index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Microverse project</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
```

Comment:
❗- Add a lang attribute to the html tag to indicate the language of the document. Example: <html lang="en"> 
❗Add a viewport meta tag to adjust the viewport on mobile devices. Example: <meta name="viewport" content="width=device-width, initial-scale=1.0">